doc/crypto/api/keys/types.rst

```diff
@@ -1482,6 +1482,80 @@ SPAKE2+ keys
     .. return:: psa_ecc_family_t
         The elliptic curve family id, if ``type`` is a supported SPAKE2+ key. Unspecified if ``type`` is not a supported SPAKE2+ key.
 
+
+Module Lattice keys
```

Comment:
We concluded for ML-DSA to have separate sections for ML-DSA and ML-KEM keys as they are not related in the way that ECC keys can work in a variety of algorithms. So this would be "Module Lattice-based Key Encapsulation keys"